CHANGELOG.unreleased.md

```diff
@@ -21,6 +21,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 - Disabled the autofill feature of the brush when using this tool to erase data. [#4729](https://github.com/scalableminds/webknossos/pull/4729)
 - The rotation buttons of the 3D-viewport no longer change the zoom. [#4750](https://github.com/scalableminds/webknossos/pull/4750)
 - Improved the performance of applying agglomerate files. [#4706](https://github.com/scalableminds/webknossos/pull/4706)
+- Moved a dataset's permission settings to the sharing tab. [#4683](https://github.com/scalableminds/webknossos/pull/4763)
```

Comment:
Sorry, I'm a bit nitpicking:

```suggestion
- In the Edit/Import Dataset form, the "Sharing" tab was renamed to "Sharing & Permissions". Also, existing permission-related settings were moved to that tab.  [#4683](https://github.com/scalableminds/webknossos/pull/4763)
```

Reply:
Fair point. It is a lot more specific! Thanks for the suggestion :)